Makefile

```diff
@@ -110,4 +110,12 @@ deinit:
 	@echo "==> Deinitializing submodules"
 	@git submodule deinit --all -f
 
+spellcheck:
+	@echo "==> Spell checking website and running fixes..."
+	@docker run \
+		-v $(pwd):/scripts \
+		--workdir=/scripts \
+		nickg/misspell:latest \
+		misspell -error -w -source=text content/
```

Comment:
@apparentlymart @nfagerlund Changed to Docker task as requested 👍

Reply:
@petems Ah ha, neat! Although... I checked it out and ran this, and it doesn't seem to do anything. :/

Reply:
Ah, $(pwd) doesnt work in makefiles, have to use `$(CURDIR)`... fixing